### DISCUSSION TITLE
Domain names not redirecting

### DISCUSSION THREAD

**Discussion by ndukachukz** [vercel/community#3414] [Help]
### Summary

Hello for some reason while hosting on Vercel my domain xample.com doesn’t redirect to the www domain www.xample.com. But when visiting www.exampledomain.com it loads the web app with no error. Any help or reason why this happens.? The two domain names seems to be configured correctly on Vercel

### Steps to Reproduce

Redirecting subdomain to full domain

**Comment by joztos** [comment]
thit stuff its happening to many people out there

**Comment by EricMcWinNer** [comment]
Hi @ndukachukz did you find a solution to this problem? Experiencing the same thing today.
